Pinta.Core/Actions/FileActions.cs

```diff
@@ -39,7 +41,14 @@ public sealed class FileActions
 	public Command Print { get; }
 
 	public event EventHandler<ModifyCompressionEventArgs>? ModifyCompression;
-	public event EventHandler<DocumentCancelEventArgs>? SaveDocument;
+
+	/// <remarks>
+	/// The returned value is
+	/// <see langword="true" /> if was save was canceled,
+	/// and
+	/// <see langword="false" /> otherwise
+	/// </remarks>
+	public event AsyncEventHandler<FileActions, DocumentCancelEventArgs>.Returning<bool>? SaveDocument;
```

Comment:
Related to the other comment, it would probably make more sense to reverse this and return true if the document was saved successfully
I found it pretty confusing to figure out which methods were supposed to be returning true vs false